<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportForm</class>
 <widget class="QWidget" name="ReportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>报表</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="moveCAD">
         <property name="text">
          <string>移动</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="restoreCAD">
         <property name="text">
          <string>还原</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QAxWidget" name="axWidget">
       <property name="control" stdset="0">
        <string>{74a777f8-7a8f-4e7c-af47-7074828086e2}</string>
       </property>
       <property name="EditGripPoint" stdset="0">
        <bool>true</bool>
       </property>
       <property name="EnableDeleteKey" stdset="0">
        <bool>true</bool>
       </property>
       <property name="EnableDrawOrder" stdset="0">
        <bool>true</bool>
       </property>
       <property name="EnableIntelliSelect" stdset="0">
        <bool>true</bool>
       </property>
       <property name="EnableMouseMoveView" stdset="0">
        <bool>true</bool>
       </property>
       <property name="IsDrawCoord" stdset="0">
        <bool>false</bool>
       </property>
       <property name="ShowCommandWindow" stdset="0">
        <bool>false</bool>
       </property>
       <property name="ShowModelBar" stdset="0">
        <bool>false</bool>
       </property>
       <property name="ShowStatusBar" stdset="0">
        <bool>false</bool>
       </property>
       <property name="ShowToolBars" stdset="0">
        <bool>false</bool>
       </property>
       <property name="ViewColor" stdset="0">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="Watermark" stdset="0">
        <string>辽宁大学</string>
       </property>
       <property name="DynToolTip" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="displayResult"/>
     </item>
     <item>
      <widget class="QPushButton" name="changeContent">
       <property name="text">
        <string>确认修改</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="startTimeLabel">
         <property name="text">
          <string>起始：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="startDate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2020</year>
           <month>10</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="endTimeLabel">
         <property name="text">
          <string>结束：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="endDate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2020</year>
           <month>10</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="clearTwinkeButton">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="queryButton">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="generateDOC">
         <property name="text">
          <string>生成文档</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="printResult">
         <property name="text">
          <string>打印</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
